package.json

```diff
@@ -36,7 +36,7 @@
     "typescript": "^4.3.5"
   },
   "scripts": {
-    "clean": "yarn lerna run clean",
+    "clean": "yarn lerna run clean --parallel",
```

Comment:
Doing clean in parallel saves about 7 seconds from the build time. Might as well do it.